src/app/src/test/kotlin/org/imperial/mrc/hint/integration/HintApplicationTests.kt

```diff
@@ -41,7 +41,7 @@ class HintApplicationTests(@Autowired val restTemplate: TestRestTemplate) : Inte
         authorize()
         val rootEntity = restTemplate.getForEntity<String>("/")
         assertThat(rootEntity.statusCode).isEqualTo(HttpStatus.OK)
-        assertThat(rootEntity.body!!).contains("<a href=\"/logout\">Logout</a>")
+        assertThat(rootEntity.body!!).doesNotContain("<title>Login</title>")
```

Comment:
This test could be reverted to previous stricter version, now that 'Logout' is being retained in the menu bar?

Reply:
The logout link lives in the vue component, not the raw html, so won't be detectable here!

Reply:
Fair point!